content/docs/hardware/v00.03.12/parts/_index.de.md

```diff
@@ -123,7 +123,7 @@ aliases:
 <tr>
   <td>1</td>
   <td>Batteriekabel und Platinenstecker (gerade, 2-polig, mit rot-schwarzen Kabeln, 2.54mm RM)</td>
-  <td><a href="https://www.reichelt.de/platinensteckverbinder-gerade-weiss-2-polig-ps-25-2g-ws-p14825.html">Link</a></td>
+  <td><a href="https://www.reichelt.de/platinensteckverbinder-gerade-weiss-2-polig-ps-25-2g-ws-p14825.html">Link1</a> <a href="https://www.pollin.de/p/buchsenleiste-serie-ps-451471">Link2a</a>/<a href="https://www.pollin.de/p/stiftleiste-serie-ps-451478">Link2b</a></td>
```

Comment:
If I see that correctly, these are the ones that "go backwards". They have a brown and black wire, instead of red and black. If you assign black=GND and therefore brown=BAT+, then the connector sits on the board with the part that goes upwards turned *away* from the polyfuse, but in the images of the build instructions, and with the black-red wires, we had to turn them the other way around. I think this will be a source of error in many cases, we'll have to fix the instructions for 00.03.12, but I'd rather have properly polarized and colored cables if we can. Could you find another part?

Reply:
When the cables were ordered in March/2021, the part on Reichelt was out of stock without a new availability date, and Pollin was the only seller I found to offer a comparable cable.
In the meantime, the part gets available again on Reichelt.

Reply:
I added an update for the build instruction to cover both variants.